infrastructure/key-vault.tf

```diff
@@ -3,43 +3,23 @@ data "azurerm_user_assigned_identity" "civil-mi" {
   resource_group_name = "managed-identities-${var.env}-rg"
 }
 
-module "civil_sdt_key_vault" {
-  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
-
-  name                        = "${var.product}-${var.component}-${var.env}"
-  product                     = var.product
-  env                         = var.env
-  object_id                   = var.jenkins_AAD_objectId
-  resource_group_name         = azurerm_resource_group.civil_sdt_rg.name
-  product_group_name          = "DTS Civil"
-  common_tags                 = local.tags
-  managed_identity_object_ids = [data.azurerm_user_assigned_identity.civil-mi.principal_id]
+data "azurerm_key_vault" "key_vault" {
+  name                = "${var.product}-${var.component}-${var.env}"
+  resource_group_name = "${var.product}-${var.component}-${var.env}"
 }
```

Comment:
Re-instate civil_sdt_key_vault module code.  This creates the civil-sdt key vault which we're using rather than the civil or civil-shared key vaults.